### ISSUE TITLE
Excessive number of failed connections from a single source (ASIM Network Session schema) default analytics rule -- results contain no src IP address

### ISSUE THREAD

**Issue by mbell85** [Azure/Azure-Sentinel#10713]
**Describe the bug**
The analytics rule "Excessive number of failed connections from a single source (ASIM Network Session schema)" is giving me results that contain a count with no source IP addresses listed. I have deployed the ASIM parsers and other analytics rules are working as expected. 

**To Reproduce**
1. Go to Azure Sentinel and configure the analytics rule (after deploying ASIM parsers). 
2. Wait for analytics rule to generate an alert and then click on the results. For more details, click on "Link to LA" under event overview.
3. See issue.

**Expected behavior**
Expect to be able to see the source IP which is causing the count to go over the programmed threshold limit.

**Screenshots**
![count-no-source](https://github.com/Azure/Azure-Sentinel/assets/43760484/4569cc02-7444-4781-b65a-ead01dd51931)

**Comment by mbell85** [comment]
No one?

**Comment by vakohl** [comment]
SrcIpAddrs coming null means that the source did not provide SrcIp in the log. Worth checking the raw log to verify if the SrcIp is coming from source or not. What is the source networksession logs in your environment?
If source is not providing SrcIp details, it is worth checking why for network failures no source details being logged.

**Comment by mbell85** [comment]
Makes sense. I'll take a look today and report back. I thought Network Events were being pulled from Azure virtual network but if it's coming from local network, we recently changed out Cisco gear to Fortinet. Might need to enable something on the FW still


--



Mark Bell
IT Administrator (CompTIA Certified Secure Infrastructure Specialist, IT Ops Specialist, A+, Network+ and Security+)
Journeys in Community Living
1130 Haley Rd.
Murfreesboro, TN 37129
615-890-4389, ext. 45<tel:+16158904389,45> (ofc)
615-295-3046<tel:+16152953046> (cell)
www.journeystn.org<http://www.journeystn.org/>
www.fb.com/journeysincommunity<http://www.fb.com/journeysincommunity>
www.twitter.com/journeystn<http://www.twitter.com/journeystn>

NOTICE: This email may contain confidential (including but not limited to) HIPAA-protected and/or privileged information intended only for specific, predetermined recipients. If you are not the intended recipient, you are hereby notified that any review, further dissemination, distribution or duplication of this communication is STRICTLY FORBIDDEN. Please delete and/or destroy all copies of this message after notifying Mark Bell of the error by reply email or calling 615-295-3046<tel:+16152953046>.

________________________________

**Comment by v-sudkharat** [comment]
@mbell85, Hope you're doing well. Did you get chance to check on shared suggestion. Thanks!

**Comment by vakohl** [comment]
@mbell85 I'm closing this issue for now since you are validating our suggestion. If you got different findings and need further help. Please re-open the issue.
CC: @v-sudkharat
